drivers/sensor/max31865/max31865.c

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2022 HAW Hamburg FTZ-DIWIP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "max31865.h"
+
+int initmax31865(const struct device *dev)
```

Comment:
All of the functions in this file should be static
```suggestion
static int init_max31865(const struct device *dev)
```